haas/api.py

```diff
@@ -605,20 +702,29 @@ def show_network(network):
 
     network = _must_find(model.Network, network)
 
-    if network.access is not None:
-        auth_backend.require_project_access(network.access)
+    if network.access:
+        #auth tests: admin owned networks with a project access
+        #TODO only network owner allowed to show the network?
```

Comment:
I don't have a very strong opinion. As long as we come to a clear decision and document it I'm happy.